docs/source/contents/4_draft_mapping.rst

```diff
@@ -531,9 +771,25 @@ typeOfResource
 classification
 ==============
 
-relateItem
+relatedItem
 ==========
```

Comment:
When you fixed the spelling, you didn't update the heading to be the same length.  This needs to be fixed.